### PR TITLE
Changes in the Car Operations Menu for the lines with brake information

### PR DESCRIPTION
**Pull request by sweiland-openrails**
Changes in the Car Operations Menu (part of the Train Operations Window):

Toggle Handbrake to set / Toggle Handbrake to off
Toggle Brake Hose to connected / Toggle Brake Hose to unconnected
Open Front Angle Cock / Close Front Angle Cock
Open Front Angle Cock / Close Rear Angle Cock
Open Front Bleed Off Value / Close Rear Bleed Off Value

Lines now show the action which will take place when pressing a line. And non valid actions will be grayed out. 

And also a small change in the BRAKE INFORMATION HUD. Where sometimes an unset handbrake is not shown as 0%, sometimes it does. Now it always does show up as 0%     (unset) or 100% (set).

see http://www.elvastower.com/forums/index.php?/topic/37062-proposal-f9-train-operations/page__p__296373__hl__%2Btrain+%2Boperations__fromsearch__1#entry296373

and https://trello.com/c/Bva4dtYW

### CODE REVIEW

**Comment by SteelFill** [comment]
I like this change, makes it much more obvious what my actions will do, but I have a few style considerations that could help.

- Capitalization is inconsistent. Normally in this menu, every word is capitalized, but the new text introduced here doesn't always follow that trend. (For example, "Toggle Handbrake to **s**et". Following the trend set, "Set" should be capitalized.
- It would be better to change the terminology to not use "toggle" as the action word when another action is simultaneously specified. ("Toggle Handbrake to Set" just sounds clunky. "Set Handbrake" is a much shorter way to say the same thing.)
- For continued consistency, it would be nice to see similar work done for the remaining commands. In particular, "Toggle MU Connection" -> "Connect/Disconnect MU Cable", "Toggle Power" -> "Switch Power On/Off", "Toggle Battery Switch" -> "Switch Battery On/Off", "Toggle Electric Train Supply Connection" -> "Connect/Disconnect Electric Train Supply Cable"

**Comment by sweiland-openrails** [comment]
hi Phillip,

Thanks for looking into this. I fully agree with your style changes. Now I have:

Unset Handbrake/Set Handbrake
Disconnect Brake Hose/Connect Brake Hose
Close Front Angle Cock/Open Front Angle Cock
Close Rear Angle Cock/Open Rear Angle Cock
Close Bleed Off Valve/Open Bleed Off Valve

The rest I'll look into after this change has been accepted. For shunting the brakes are important. The rest is less. And the results for these actions are less clear, I guess. Still have to look into this.

**Comment by sweiland-openrails** [comment]
I'm always a bit confused about the Brake Hose entry in the Car Operations Menu, is it in front or at the rear of the car? Therefore I've changed this entry into two entries: Connect Front Brake Hose and Connect Rear Brake Hose. Just the menu changed, no data changes. If for the above example you click Disconnect Rear Brake Hose the brake hose between 0-0 and 0-1 gets disconnected. Seems easier to me.

**Review by cjakeman — COMMENTED** [review]
Hi Siebren,

I've approved your Trello card, so any dev is free now to review, approve and merge this PR.

Thanks, 
Chris

**Comment by sweiland-openrails** [comment]
hi Phillip,

Since you know already about this change, could you review my pull request?

thanks, Siebren.

File: Source/Documentation/Manual/driving.rst [hunk 1]
```diff
@@ -537,13 +537,16 @@ By clicking in this menu it is possible:
   for both electric and diesel locomotives;
 - to connect or disconnect locomotive operation with that of the player 
   locomotive;
-- to connect or disconnect the car's air hoses from the rest of the consist;
-- to toggle the angle cocks on the air hoses at either end of the car 
-  between open and closed;
-- to toggle the bleed valve on the car to vent the air pressure from the 
+- to connect of disconnect the battery;
```
[the remark this answers is not in the document]
Comment (sweiland-openrails) [reply Source/Documentation/Manual/driving.rst]: Changed the documentation accordingly. Thanks for looking into this,

**Review by SteelFill — APPROVED** [review]
Looks good now, and as a new OpenRails team member, I can approve your changes.

Approved

**Comment by sweiland-openrails** [comment]
thanks Phillip and good to have you in the team.

**Pull request merged**